client/src/app/App.tsx

```diff
@@ -259,7 +261,7 @@ class App extends Component<RouteComponentProps<AppRouteProps> & { config: IConf
                                             />
                                             <Route
                                                 path="/:network/address/:hash"
-                                                component={(props: RouteComponentProps<AddressRouteProps>) =>
+                                                component={(props: RouteComponentProps<OgAddressRouteProps>) =>
                                                 (
```

Comment:
Why `Og` ?

Reply:
We have multiple AddressRouteProps imported above so it's named to be clear that is for OG network. From imports: `import { AddressRouteProps as OgAddressRouteProps } from "./routes/og/AddressRouteProps";`